packages/data/src/world/oot/gerudo_training_grounds.yml

```diff
@@ -91,17 +91,17 @@
 "Gerudo Training Grounds Water":
   dungeon: GTG
   exits:
-    "Gerudo Training Grounds Underwater": "can_play_time && has_tunic_zora && has_iron_boots"
+    "Gerudo Training Grounds Underwater": "can_play_time && has_iron_boots && (has_tunic_zora || longshot_anywhere)"
```

Comment:
what i meant is without swimming it hard requires the tunic or longshot anywhere to get the chest.. Also, i don't know which number it would be but only one of the silver rupees are on the ground and wouldn't need anything. The ceiling rupee still needs hookshot anywhere for it too.

I would just revert this line for now, and then i'll handle a silver rupee event since it will be easier, and I was going to do events for them all anyway.